proxy/Dockerfile

```diff
@@ -4,24 +4,18 @@ FROM python:3.10.12-slim-buster
 ENV PYTHONDONTWRITEBYTECODE 1
 ENV PYTHONUNBUFFERED 1
 
-RUN apt-get -y update && apt-get install  --no-install-recommends -y iptables=1.8.2-4 sudo=1.8.27-1+deb10u6 &&\
-    apt-get clean &&\
-    rm -rf /var/lib/apt/lists/*
-
-RUN pip install --upgrade pip==24.0 --no-cache-dir
-
-RUN adduser --disabled-password --gecos '' worker
-RUN adduser worker sudo
-RUN echo '%sudo ALL=(ALL) NOPASSWD:ALL' >> /etc/sudoers
+RUN  adduser --disabled-password --gecos '' worker
 
 USER worker
 WORKDIR /home/worker
 
-COPY --chown=worker:worker proxy/requirements.txt .
-RUN pip install -r requirements.txt --no-cache-dir
-
 ENV PATH="/home/worker/.local/bin:${PATH}"
 
+COPY --chown=worker:worker proxy/requirements.txt .
+# hadolint ignore=DL3013
+RUN pip install --upgrade pip --no-cache-dir &&\
```

Comment:
@akihikokuroda @psschwei I would like to know your opinion here. Do you think is a good idea ignore the version when upgrading pip? Instead of:

```py
RUN pip install --upgrade pip==24.0 --no-cache-dir
```

Thinking in that we are going to need to review this version from time to time.

Reply:
we just posted at the same time, @akihikokuroda 😂

Reply:
my two cents: if the old version of pip isn't causing problems, I wouldn't worry about upgrading it (there was a case a while back where version resolution through an old pip was very slow, but I don't think that's impacting the default pip in python 3.10...)

Reply:
Yep, and less in this case where dependencies are very small. I like your idea @psschwei , in case someone sees a problem let us know!